### PR TITLE
v4 - .c-select shouldn't inherit white text color

### PR DESCRIPTION
**Pull request by zalog**
This:

``` html
<div class="bg-info">
    <select class="c-select">
        <option selected>Open this select menu</option>
        <option value="1">One</option>
        <option value="2">Two</option>
        <option value="3">Three</option>
    </select>
</div>
```

render this:

### CODE REVIEW

**Comment by zqw** [comment]
:rabbit2:

File: scss/_custom-forms.scss [hunk 1]
```diff
@@ -135,6 +135,7 @@
   padding: .375rem 1.75rem .375rem .75rem;
   padding-right: .75rem \9;
   vertical-align: middle;
+  color: $input-color;
```
Comment (zqw) [inline scss/_custom-forms.scss]: good

**Comment by XhmikosR** [comment]
@zalog: please fix the tests.

**Comment by zalog** [comment]
Hmm, It's ok now?

**Comment by XhmikosR** [comment]
It's OK now.

/CC @cvrebert @mdo

**Pull request merged**